### DISCUSSION TITLE
Sequelize DB integration issue with sentry (Performance Monitoring)

### DISCUSSION THREAD

**Discussion by Bhavin-jumpbyte** [sequelize/sequelize#14728] [Help & Questions]
## Issue Description

I'm using sequelize as DB and MySQL dialect, which internally use the mysql2 package. I need some help to start so I can integrate Sequelize DB integration with Sentry for performance monitoring. Unfortunately, they (Sentry) don't provide integrations for Sequelize or mysql2. I tried to modify the @sentry/tracing package code manually to support mysql2. But unfortunately, it didn't work.

```
    Sentry.init({
      dsn,
      integrations: [
        new Sentry.Integrations.Http({ tracing: true }),
        new SentryTracing.Integrations.Express({ app })
      ],
      tracesSampleRate: 1.0
    });
    
    app.use(Sentry.Handlers.requestHandler());
    app.use(Sentry.Handlers.tracingHandler());
    app.use(Sentry.Handlers.errorHandler());
```

## What do you expect to happen?
```
new SentryTracing.Integrations.Sequelize() OR new SentryTracing.Integrations.MySql2()
```
## Environment

```
   Sequelize Version: 5.22.4
   Node Version: 14.18.0,
   @sentry/node Version:  7.5.0,
   @sentry/tracing Version:  7.5.0
```

**Comment by mrefaie** [comment]
I have the same problem with Postgres and Sequelize

**Reply by shijo-keyvalue** [reply]
@Bhavin-jumpbyte 
@mrefaie 
Is this issue resolved ?
I am also facing the same issue. Database spans are not shown in Sentry performance dashboad , while using sequelize ORM (with mysql2) in an express server.

Edit :
Finally found a workaround for the same. If we use opentelemetry along with sentry , then we are able to get the database spans.
And sentry team updated that its working on a node-sdk version where mysql2 will be supported out of the box.
https://github.com/getsentry/sentry-javascript/issues/3433#issuecomment-1403935160
